kinesis_manager/CMakeLists.txt

```diff
@@ -9,23 +9,20 @@ if(AWSSDK_FOUND)
   AWSSDK_DETERMINE_LIBS_TO_LINK(SERVICE AWSSDK_LIBS_TO_LINK)
   link_directories("${AWSSDK_LIB_DIR}")
 endif()
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/") # Required for FindKVSSDK
 
 find_package(CURL REQUIRED)
-find_package(KVSSDK REQUIRED)
+
+## Add KVS SDK libraries
+include(FindKVSSDK.cmake)
```

Comment:
I think FindKVSSDK.cmake is used more like an `include()` file than a `find_package()` module, because it is setting some variables needed by the top-level CMakeLists.txt regardless of whether or not the Kinesis Producer Library was found. Using it like an `include()` file also avoids the need to muck around with the `CMAKE_MODULE_PATH` variable.